index.html

```diff
@@ -30,16 +40,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
```

Comment:
Good job giving it a different class name! way easier solution to using css nth child selectors!